Source/XUnitExtensions/Lit/ShellLitCommand.cs

```diff
@@ -37,7 +37,14 @@ public class ShellLitCommand : ILitCommand {
 
       process.StartInfo.EnvironmentVariables.Clear();
       foreach (var passthrough in passthroughEnvironmentVariables) {
-        process.StartInfo.EnvironmentVariables.Add(passthrough, Environment.GetEnvironmentVariable(passthrough));
+        if (passthrough.Contains('=')) {
```

Comment:
It's the "variable" `TEST_DAFNY=true`, which is technically not an environment variable, it's a trick so that I don't need to set the environment variable and still set an environment variable to the child process.
I'm getting rid of this trick in the next PR where compilers won't be aware anymore that we are in test mode.